eve/main.yml

```diff
@@ -177,22 +177,30 @@ stages:
           usePTY: true
           haltOnFailure: false
 
-  single-node:
+  single-node-downgrade-test:
     worker:
       type: openstack
       image: CentOS-7-x86_64-GenericCloud-1809.qcow2
       flavor: m1.large
       path: eve/workers/openstack-single-node
     steps:
       - Git: *git_pull
+      - SetPropertyFromCommand:
```

Comment:
Please use yaml tag (`&set_prev_vers` `*set_prev_vers`) for all common step for downgrade/upgrade to remove copy paste. At least :
- `Set previous version`
- `Retrieve ISO version N-1 image`
- `Retrieve ISO N-1 image checksum`
- `Check N-1 image with checksum`

Reply:
@TeddyAndrieux lets merge one first and then we can add and re-use `anchors` in the second PR. This is the reason why the second PR on `Upgrade` is in draft mode till now

Reply:
Ok yes you can add/use them in a single PR, as you want

Reply:
@sayf-eddine-scality please do this, as the upgrade merged now :)